TestingApp/ViewController.swift

```diff
@@ -161,12 +161,34 @@ extension ViewController {
 
     func updateConfiguration(with queryItems: [URLQueryItem]) {
         Configuration(queryItems: queryItems).map { configuration = $0 }
-        queryItems.first(where: { $0.name == "queue_id"})?.value.map {
+        queryItems.first(where: { $0.name == "queue_id" })?.value.map {
             queueId = $0
         }
     }
 
     func applyRemoteConfig(with name: String) {
```

Comment:
Just to clarify, what does this function do exactly?

Reply:
When `AppDelegate` catches deeplink, it calls `updateConfiguration `  func. The last one initialise `Configuration` from link query and stores new config in local property. 
You can configure the SDK either through Settings screen or deeplink.

Reply:
4db2d612c6b126a7284e64b0076b8f19e562290a